src/rpc/core_rpc_server.cpp

```diff
@@ -1507,7 +1553,7 @@ namespace cryptonote::rpc {
     get_block.response["block_header"] = header;
     std::vector<std::string> tx_hashes;
     tx_hashes.reserve(blk.tx_hashes.size());
-    std::transform(blk.tx_hashes.begin(), blk.tx_hashes.end(), tx_hashes.begin(), [](const auto& x) { return tools::type_to_hex(x); });
+    std::transform(blk.tx_hashes.begin(), blk.tx_hashes.end(), std::back_inserter(tx_hashes), [](const auto& x) { return tools::type_to_hex(x); });
     get_block.response["tx_hashes"] = tx_hashes;
```

Comment:
`std::move`?

Reply:
Changed, though it probably doesn't do anything because of the vector<string> -> json conversion going on here.

Reply:
fair enough, but shouldn't hurt